src/lib/libpthread.js

```diff
@@ -1199,6 +1199,16 @@ var LibraryPThread = {
                         '_emscripten_check_mailbox',
                         '_emscripten_thread_mailbox_await'],
   $checkMailbox: () => {
+    // The mailbox is checked using a setTimeout() handler, but the application
+    // may have quit (or crashed) after the setTimeout() was registered,
+    // so first check if that might be the case, and no-op if so.
+#if EXIT_RUNTIME
+    if (runtimeExited) return;
+#endif
+#if !MINIMAL_RUNTIME
+    if (ABORT) return;
```

Comment:
Do we ever need both this checks?  i.e. should this be an `elif` ?

Reply:
I don't think so.

Both booleans are used elsewhere in code, so I think it is correct to be symmetric here as well.